tests/test_benchmark_simple_scenarios.py

```diff
@@ -93,46 +93,22 @@ def test_benchmark_AE_grid_battery(self, margs):
     )
     @mock.patch("argparse.ArgumentParser.parse_args", return_value=argparse.Namespace())
     def test_benchmark_ABE_grid_pv_bat(self, margs):
-        use_case = "AB"
-        main(
-            path_input_folder=os.path.join(TEST_INPUT_PATH, use_case),
-            input_type=CSV_EXT,
-            path_output_folder=os.path.join(TEST_OUTPUT_PATH, use_case),
-        )
-        #with open(
-            #os.path.join(TEST_OUTPUT_PATH, use_case, "json_with_results.json")
-        #) as fp:
-            #AB_dict_values = convert_special_types(json.load(fp))
-
-        AB_busses_flow = pd.read_excel(
-            os.path.join(TEST_OUTPUT_PATH, use_case, "timeseries_all_busses.xlsx"),
-            sheet_name="Electricity bus",
-        )
-        #AB_busses_flow = AB_busses_flow.set_index("Unnamed: 0")
-        # compute the sum of the excess electricity
-        AB_busses_flow["Electricity excess_sink"] = AB_busses_flow.sum()
-
-        use_case = "ABE"
-        main(
-            path_input_folder=os.path.join(TEST_INPUT_PATH, use_case),
-            input_type=CSV_EXT,
-            path_output_folder=os.path.join(TEST_OUTPUT_PATH, use_case),
-        )
-        #with open(
-            #os.path.join(TEST_OUTPUT_PATH, use_case, "json_with_results.json")
-        #) as fp:
-            #ABE_dict_values = convert_special_types(json.load(fp))
-
-        ABE_busses_flow = pd.read_excel(
-            os.path.join(TEST_OUTPUT_PATH, use_case, "timeseries_all_busses.xlsx"),
-            sheet_name="Electricity bus",
-        )
-        #ABE_busses_flow = ABE_busses_flow.set_index("Unnamed: 0")
-        # compute the sum of the excess electricity
-        ABE_busses_flow["Electricity excess_sink"] = ABE_busses_flow.sum()
-
-        assert AB_busses_flow["Electricity excess_sink"] > ABE_busses_flow["Electricity excess_sink"]
-
+        use_case = ["AB", "ABE"]
```

Comment:
Maybe a docstring explaining what the test does with words would be good :)

Reply:
okay will add them now